src/ucp/wireup/select.c

```diff
@@ -580,10 +587,12 @@ ucp_wireup_add_lane(const ucp_wireup_select_params_t *select_params,
                                                       remote_event_flags);
     }
 
-    dst_md_index = select_params->address->address_list
-                                [select_info->addr_index].md_index;
-    return ucp_wireup_add_lane_desc(select_info, dst_md_index, lane_type,
-                                    is_proxy, select_ctx);
+    dst_md_index  = select_params->address->address_list
+                                 [select_info->addr_index].md_index;
+    dst_dev_index = select_params->address->address_list
+                                 [select_info->addr_index].dev_index;
```

Comment:
minor: can create local pointer to `select_params->address->address_list` or even its entry

Reply:
fixed